SIT.Manager.Avalonia/Views/MainView.axaml

```diff
@@ -55,7 +55,7 @@
 		<ui:NavigationView PaneDisplayMode="Left"
 						   IsBackButtonVisible="False"
 						   IsPaneToggleButtonVisible="False"
-						   OpenPaneLength="150"
+						   OpenPaneLength="170"
```

Comment:
Teensy lil thing, can we swap this to 160? Seems to still avoid cutting the text off but is just a little smaller ^^

Reply:
160 doesn't work for Ukrainian. :(

Reply:
167 is the max.

Reply:
166 :)